charts/images.yaml

```diff
@@ -29,14 +29,14 @@ images:
   tag: "v1.20.10"
   targetVersion: "1.20.x"
 - name: cloud-controller-manager
-  sourceRepository: github.com/gardener/cloud-provider-azure
-  repository: eu.gcr.io/gardener-project/kubernetes/cloud-provider-azure
-  tag: "v1.21.4"
+  sourceRepository: https://github.com/kubernetes-sigs/cloud-provider-azure
+  repository: mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager
+  tag: "v1.1.1"
   targetVersion: "1.21.x"
 - name: cloud-controller-manager
-  sourceRepository: github.com/gardener/cloud-provider-azure
-  repository: eu.gcr.io/gardener-project/kubernetes/cloud-provider-azure
-  tag: "v1.22.0"
+  sourceRepository: https://github.com/kubernetes-sigs/cloud-provider-azure
+  repository: mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager
+  tag: "v1.1.1"
   targetVersion: ">= 1.22"
```

Comment:
```suggestion
  sourceRepository: github.com/kubernetes-sigs/cloud-provider-azure
  repository: mcr.microsoft.com/oss/kubernetes/azure-cloud-controller-manager
  tag: "v1.1.1"
  targetVersion: ">= 1.21"
```

Reply:
If I read the [matrix](https://github.com/kubernetes-sigs/cloud-provider-azure#current-status) correctly, it should be safe to use for version `v1.22` and higher. For v1.21 probably we should use `v1.0.5` of the image.

The out-of-tree CCM should basically be a superset of the in-tree. The in-tree only gets bug fixes since v1.20 and these are first implemented in the out-of-tree then ported back.

Reply:
Both versions work for 1.21 but the required parts for using the cluster on vmss-flex was also part of `v1.0.5`

Reply:
I will change `image.yaml` to use the `v1.0.5` for k8s `1.21`

Reply:
all tests are working with `v1.0.5` and k8s `1.21` I updated the pr